.github/workflows/release.yml

```diff
@@ -35,6 +35,24 @@ jobs:
           checksum_file=$(echo "$ARTIFACTS" | jq -r '.[] | select (.type=="Checksum") | .path')
           echo "::set-output name=hashes::$(cat $checksum_file | base64 -w0)"
 
+  publish:
+    # outputs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: git fetch --prune --unshallow
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+          check-latest: true
+      - uses: imjasonh/setup-ko@v0.6
+      - name: push image on ghcr
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          tag=$(echo ${{ github.ref }} | cut -c11-) # get tag name without tags/refs/ prefix.
+          ko push --base-import-paths -t ${tag} .
```

Comment:
I think `ko push` is incorrect, and we probably want to adjust this either way:
https://github.com/ko-build/ko/blob/e23aebe18bc2ba0906a500da8592ce60b14668c4/.github/workflows/image.yaml#L33

Reply:
@mattmoor you are right 🤦🏼 it should be `publish` or `build` indeed ! I'll update 👼🏼

Reply:
However, I do want to tag it with the tag name (0.12.0) versus the sha (commit id)

Reply:
Sure, I was thinking we'd commit bash crimes to get a list into `--tags`. 😉 

Really the main thing I'm worried about is that we're not signing these images, which the other flow is doing.

Reply:
@mattmoor indeed, that would have been my follow-up question 🙃 What would be the flow to sign those images as well ? 👼🏼